doc/server-secrets.md

```diff
@@ -4,26 +4,33 @@ It is possible to provide a token or credentials for a number of external
 services. These may be used to lift a rate limit or provide access to
 private resources from a self-hosted instance.
 
-Secrets can be set in  `private/secret.json`. For example:
+There are two ways of setting secrets:
```

Comment:
I missed this in #2599 but the Azure DevOps token should be added to this list (and corresponding config locations).

Reply:
The token name is `azure_devops_token`

Reply:
I can update this doc with the relevant info/docs if you'd like, but not sure where all else the token would need to be added in the other changed files.

Reply:
That'd be great! Besides the doc, it needs to be added to the config schema in `lib/server.js` and have an entry added with the environment variable mapping in `custom-environment-variables.yml`.

Probably _that_ ^^ should be documented somewhere too.

Reply:
Definitely! Thanks

Reply:
Done! I feel pretty confident the Azure DevOps content in `server-secrets.md` will make sense to Azure DevOps users, hope it's clear enough for new users too